mindsdb/integrations/handlers/gmail_handler/README.md

```diff
@@ -0,0 +1,95 @@
+# Gmail API Integration
+
+This handler integrates with the [Gmail API](https://developers.google.com/gmail/api/guides)
+to make gmail data available to use for model training and predictions.
+
+## Example: Write automated emails
+
+To see how the Gmail handler is used, let's walk through a simple example to create a model to predict
+
+## Connect to the Gmail API
+## Prequisites
+*  You will need to have a Google account and have enabled the Gmail API.
+*  A project in the [Google Cloud Console](https://console.cloud.google.com/) with Gmail Api enabled.
+* A credentials file for the project. You can find more information on how to do this [here](https://developers.google.com/workspace/guides/create-credentials).
+
+We start by creating a database to connect to the Gmail API. In order to do this,as said before, you will need to obtain credentials:
+
+
+**Optional:**  The credentials file can be stored in the gmail_handler folder in
+the [mindsdb/integrations/gmail_handler](mindsdb/integrations/handlers/gmail_handler) directory.
+
+~~~~sql
+CREATE
+DATABASE  gmail_test
+WITH  ENGINE = 'gmail',
+parameters = {
+    "path_to_credentials_file": "/home/marios/PycharmProjects/mindsdb/mindsdb/integrations/handlers/gmail_handler/credentials.json"
```

Comment:
can we make this that it supports a path to s3 signed url?

Reply:
yes of course!I will work on that!

Reply:
i added the support of s3 signed url.You can check that and give me feedback.